### PR TITLE
fix(tests): make tests support Firefox 53

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa-content-server/issues/4971

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,15 +106,15 @@
...
+    "intern": "3.4.3",
     "npmshrink": "1.0.1",
     "proxyquire": "1.7.4",
     "sinon": "1.17.3",
     "sync-exec": "0.6.2",
     "xmlhttprequest": "git://github.com/zaach/node-XMLHttpRequest.git#onerror"
   },
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=4.8.0"
```
Comment (shane-tomlinson) [inline package.json]: he.. How'd we never do this before? Will this cause any blowups on any outstanding fxa-dev boxes that don't yet have node 4.8?

Comment (vladikoff) [inline package.json]: Nothing really uses this property, npm mostly just complains when there is a mismatch

**Comment by vladikoff** [comment]
Some strange `‣ Created session firefox on any platform (00b6cc88-6176-43b0-a8f2-7a8c6aced22d)
× firefox on any platform - mocha tests - run the mocha tests (0.019s)
Error: [POST http://localhost:4444/wd/hub/session/00b6cc88-6176-43b0-a8f2-7a8c6aced22d/timeouts / {"type":"implicit","ms":300000}] Missing 'type' parameter
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'` 

in node 6.10.2 on Travis only

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 32cc858b6f9bba2502e30293ce497ec351b5ccb0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by shane-tomlinson** [comment]
> Some strange ‣ Created session firefox on any platform (00b6cc88-6176-43b0-a8f2-7a8c6aced22d) × firefox on any platform - mocha tests - run the mocha tests (0.019s) Error: [POST http://localhost:4444/wd/hub/session/00b6cc88-6176-43b0-a8f2-7a8c6aced22d/timeouts / {"type":"implicit","ms":300000}] Missing 'type' parameter Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'

in node 6.10.2 on Travis only

I was getting this locally too, when using Fx 50 instead of 53.

**Comment by shane-tomlinson** [comment]
> Suite firefox on any platform - oauth settings clients FAILED
UnknownError: [POST http://localhost:4444/wd/hub/session/ca3c0e36-7cb1-4e56-a069-12c590e9cdf7/timeouts / {"type":"implicit","ms":28000}] Not a Number
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'stomlinson-30887', ip: '10.239.33.137', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.4', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{rotatable=false, raisesAccessibilityExceptions=false, appBuildId=20161104212021, version=50.0, platform=MAC, proxy=Proxy(), command_id=1.0, specificationLevel=0.0, acceptSslCerts=false, processId=55840.0, browserVersion=50.0, platformVersion=16.5.0, XULappId={ec8030f7-c20a-464f-9b0e-13a3a9e97384}, browserName=firefox, takesScreenshot=true, takesElementScreenshot=true, javascriptEnabled=true, platformName=darwin}]
Session ID: dc9c3189-8d9c-e04b-89d7-da9b342d1776
  at runRequest  <node_modules/intern/node_modules/leadfoot/Session.js:92:40>
  at <node_modules/intern/node_modules/leadfoot/Session.js:113:39>
  at new Promise  <node_modules/intern/node_modules/dojo/Promise.ts:411:3>
  at ProxiedSession._post  <node_modules/intern/node_modules/leadfoot/Session.js:67:10>
  at ProxiedSession.Session.setTimeout  <node_modules/intern/node_modules/leadfoot/Session.js:540:22>
  at ProxiedSession.<anonymous>  <node_modules/intern/node_modules/leadfoot/Session.js:2549:16>
  at ProxiedSession.setFindTimeout  <node_modules/intern/node_modules/dojo/lang.ts:85:12>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:635:19>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at Command.target.(anonymous function) [as setFindTimeout]  <node_modules/intern/node_modules/leadfoot/Command.js:610:11>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:268:8>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:55:25>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:489:9>
  at process._tickCallback  <node.js:418:13>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:368:14>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:489:9>
  at process._tickCallback  <node.js:418:13>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:365:8>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:55:25>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:489:9>
  at process._tickCallback  <node.js:418:13>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Suite.registerSuite.beforeEach  <tests/functional/oauth_settings_clients.js:39:10>
  at Suite.target.(anonymous function).dispatcher [as beforeEach]  <node_modules/intern/browser_modules/dojo/aspect.ts:102:35>
  at Promise.catch.suite.async  <node_modules/intern/lib/Suite.js:237:51>
  at new Promise  <node_modules/intern/browser_modules/dojo/Promise.ts:411:3>
  at runLifecycleMethod  <node_modules/intern/lib/Suite.js:219:12>
  at runWithCatch  <node_modules/intern/lib/Suite.js:349:15>
  at next  <node_modules/intern/lib/Suite.js:352:17>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>

**Pull request merged**